src/rosecommon/packets/srv_skill_damage.xml

```diff
@@ -0,0 +1,26 @@
+<packet ePacketType = "PAKWC_SKILL_DAMAGE">
+    <includeXml path = "item.xml" />
+
+    <element name = "target" type = "uint16_t" />
+    <element name = "source" type = "uint16_t" />
+    <element name = "id" type = "uint16_t" />
+    <element name = "amount" type = "float" />
+    <!-- STB state id for this skill, only first two bits are used (0x00, 0x01, 0x02, 0x03)-->
+    <element name = "state" type = "uint8_t" />
+
+    <element name = "damage" type = "uint32_t" />
+    <element name = "action" type = "DamageAction" />
```

Comment:
`action` is a bitfield. This may cause issues since this value can be a combo of different values that would not match the enum defined below.